frontend/ui/src/app/components/dashboard/dashboard.component.html

```diff
@@ -0,0 +1,39 @@
+<div class="p-4 sm:ml-64 bg-gray-50 dark:bg-gray-900">
+  <div class="p-4">
+    @if ((deploymentTargets$ | async)?.length === 0) {
+      <div
+        class="p-4 mb-4 text-sm text-blue-800 max-w-128 mx-auto rounded-lg bg-blue-50 dark:bg-gray-800 dark:text-blue-400"
+        role="alert">
+        Click <a class="cursor-pointer underline" (click)="openWizard()">here</a> to continue with the onboarding
+        wizard.
+      </div>
+    }
+
+    <div class="flex flex-col gap-4">
+      <h1 class="text-3xl font-extrabold leading-none tracking-tight text-gray-900 md:text-4xl dark:text-white">
```

Comment:
should be `h2` I think: https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/Heading_Elements#avoid_using_multiple_h1_elements_on_one_page